### DISCUSSION TITLE
cannot install the "with-react-intl" example project?

### DISCUSSION THREAD

**Discussion by shervinchen** [vercel/next.js#41242] [Help] [answered]
when i run the code `pnpm create next-app --example with-react-intl with-react-intl-app` to install the project, it returns me the error 
```
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies
.
└─┬ eslint-plugin-formatjs
  └── ✕ missing peer eslint@"7 || 8"
Peer dependencies that should be installed:
  eslint@"7 || 8"
```
Is this because of a dependency problem?can anyone fix it ?

**Comment by icyJoseph** [comment, marked as answer]
This is a matter of, almost, preference, because peer-deps are not installed by default by `pnpm`, so you could create a `.npmrc` file and follow either of these suggestions:

- hint: If you want peer dependencies to be automatically installed, add "auto-install-peers=true" to an .npmrc file at the root of your project.
- hint: If you don't want pnpm to fail on peer dependency issues, add "strict-peer-dependencies=false" to an .npmrc file at the root of your project.

However, I do see that other examples do have `eslint` installed. I could make an MR to include `eslint` explicitly, when I get some free time. Or if you feel like fixing it yourself, I think you should do it.

The arguments to do it would be:

- `out-of-box` installation, without tweaking `pnpm`'s defaults
- reducing implicit dependencies
- alignment with other examples that explicitly install `eslint`
